content/10.diagnostics.md

```diff
@@ -173,4 +173,5 @@ Specifically, the individuals at risk include:
 
 However, this method of testing administration does not detect a high proportion of infections and does not allow for test-and-trace methods to be used.
 Individuals who are asymptomatic (i.e. potential spreaders) and individuals who are able to recover at home are therefore often unaware of their status.
-For instance, a recent study from the Imperial College estimates that in Italy the true number of infections is around 5.9 million against the 70,000 detected as of March 28th [@doi:10.25561/77731].
+For instance, a recent study from Imperial College estimates that in Italy the true number of infections is around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28th [@doi:10.25561/77731].
+Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population ~20 million, corresponding to ~1.5% of the population, but up to ~20% of the population tested positive when evaluated with antibody tests [@url:ny.governor.gov/nyforwardbook].
```

Comment:
```suggestion
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million, corresponding to ~1.5% of the population, but 12% of test individuals statewide and up to ~20% of the population in some places tested positive when evaluated with antibody tests [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
```

Reply:
@rando2 to make the last clause slightly clearer, maybe rechange the order to: "but when evaluated with antibody tests ~12% of individuals were positive statewide and up to ~20% in some places were positive" and then the correct url reference. (reason for clarifying: right now the modified clause is ambiguous with respect to what the ~12% means exactly)